plugins/aws/ec2/openCustomPorts.js

```diff
@@ -0,0 +1,91 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Open Custom Ports',
+    category: 'EC2',
+    description: 'Ensures that the defined ports are not exposed publicly',
+    more_info: 'Security groups should be used to restrict access to ports from known networks.',
+    link: 'https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html',
+    recommended_action: 'Modify the security group to ensure the ports are not exposed publicly.',
+    apis: ['EC2:describeSecurityGroups'],
+    settings: {
+        open_port_allowed_list: {
+            name: 'EC2 Allowed Open Ports',
+            description: 'A comma-delimited list of ports that indicates open ports allowed for any connection',
+            regex: '[a-zA-Z0-9,]',
+            default: [80, 443]
+        }
+    },
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var allowed_open_ports = this.settings.open_port_allowed_list.default;
```

Comment:
https://github.com/cloudsploit/scans/blob/master/plugins/aws/ec2/vpcElasticIpLimit.js

Reply:
`var config = {
            vpc_elastic_ip_percentage_fail: settings.vpc_elastic_ip_percentage_fail || this.settings.vpc_elastic_ip_percentage_fail.default,
            vpc_elastic_ip_percentage_warn: settings.vpc_elastic_ip_percentage_warn || this.settings.vpc_elastic_ip_percentage_warn.default
        };`